packages/zwave-js/src/lib/serialapi/capability/LongRangeSetupMessages.ts

```diff
@@ -0,0 +1,169 @@
+import { MessagePriority, encodeBitMask, parseBitMask } from "@zwave-js/core";
+import type { ZWaveHost } from "@zwave-js/host";
+import type { SuccessIndicator } from "@zwave-js/serial";
+import {
+	FunctionType,
+	Message,
+	type MessageBaseOptions,
+	type MessageDeserializationOptions,
+	MessageType,
+	expectedResponse,
+	gotDeserializationOptions,
+	messageTypes,
+	priority,
+} from "@zwave-js/serial";
+
+// TODO: move to header?
```

Comment:
which other header should this sort of thing live in, if any?

Reply:
Maybe in `packages/core/src/capabilities/Protocols.ts`, or a new LR-related file in that folder